compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Utils/AMDAIEDmaUtils.cpp

```diff
@@ -511,6 +480,91 @@ LogicalResult moveNpuDmaSyncUsersAfterAncestorInSameBlock(
   return success();
 }
 
+/// Try to add `offsetToMerge` to one of the offsets in `offsets`.  This
+/// function assumes an effective stride for `offsetToMerge` of 1one so to add
```

Comment:
I'm not sure what it should be. 1one -> one?

Reply:
eleven ;) 
I'll fix, thanks.